db/plain_table_db_test.cc

```diff
@@ -428,7 +428,7 @@ TEST_P(PlainTableDBTest, BadOptions2) {
   // Build without a prefix extractor
   // (apparently works even if hash_table_ratio > 0)
   ASSERT_OK(Put("1000000000000foo", "v1"));
-  dbfull()->TEST_FlushMemTable();
+  ASSERT_NOK(dbfull()->TEST_FlushMemTable());
```

Comment:
Why is this NOK?  Can you add a comment on why this is expected to fail?

Reply:
It uses a bad options, so the flush is expected to fail. I will add a comment